cob_base_velocity_smoother/CMakeLists.txt

```diff
@@ -1,8 +1,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(cob_base_velocity_smoother)
 
-find_package(catkin REQUIRED COMPONENTS std_msgs roscpp geometry_msgs)
-
+find_package(catkin REQUIRED COMPONENTS roscpp std_msgs geometry_msgs)
```

Comment:
Haven't recognized it in the PR at cob_driver...
This doesn't seem to be ordered alphabetically. Did this come from roscompile as well?